lab1/myswitch_fifo.py

```diff
@@ -0,0 +1,77 @@
+from switchyard.lib.userlib import *
+
+
+class ForwaridngTable:
```

Comment:
Should be "ForwardingTable" (same transpose in comment & when used in main()

Reply:
Corrected now.